library/sentry

```diff
@@ -4,12 +4,12 @@
 7.7: git://github.com/getsentry/docker-sentry@e60211c46e9fd4b4f9ee679946bc9915ae2bf0c0 7.7
 7: git://github.com/getsentry/docker-sentry@e60211c46e9fd4b4f9ee679946bc9915ae2bf0c0 7.7
 
-8.0.2: git://github.com/getsentry/docker-sentry@e9cf37071e700b6e94427a87edddd438933fc87c 8.0
-8.0: git://github.com/getsentry/docker-sentry@e9cf37071e700b6e94427a87edddd438933fc87c 8.0
-8: git://github.com/getsentry/docker-sentry@e9cf37071e700b6e94427a87edddd438933fc87c 8.0
-latest: git://github.com/getsentry/docker-sentry@e9cf37071e700b6e94427a87edddd438933fc87c 8.0
+8.0.3: git://github.com/getsentry/docker-sentry@f66fed3c2fa19086bd268117ce7262a93576718d 8.0
+8.0: git://github.com/getsentry/docker-sentry@f66fed3c2fa19086bd268117ce7262a93576718d 8.0
+8: git://github.com/getsentry/docker-sentry@f66fed3c2fa19086bd268117ce7262a93576718d 8.0
+latest: git://github.com/getsentry/docker-sentry@f66fed3c2fa19086bd268117ce7262a93576718d 8.0
 
-8.0.2-onbuild: git://github.com/getsentry/docker-sentry@e60211c46e9fd4b4f9ee679946bc9915ae2bf0c0 8.0/onbuild
+8.0.3-onbuild: git://github.com/getsentry/docker-sentry@e60211c46e9fd4b4f9ee679946bc9915ae2bf0c0 8.0/onbuild
```

Comment:
Nope, that is fine. The generate script is checking last git commit that modified the files within the directory.  The image will be rebuilt since it is `FROM sentry:8.0`.